### ISSUE TITLE
Unable to make cobweb.min.js from source

### ISSUE THREAD

**Issue by vincentmarchetti** [create3000/cobweb#4]
I am unable to compile (using the jam directives as specified in Makefile) the cobweb.min.js file; get error message that a file cobweb.js/cobweb/Browser/Core/TextureQuality.js is not found -- and indeed, there is no such file in the repository. Is the make-Makefile path for creating product cobweb.min.js not supported?

This is terminal output:
```
radium:cobweb.js vmarchetti$ make
jam compile --wrap --no-minify cobweb.js -i cobweb.js -o cobweb.min.js
compiling cobweb.min.js
include cobweb.js
Error: Error: ENOENT: no such file or directory, open '/Users/vmarchetti/Desktop/SPRI X3D/SPRI_2015-11/COBWEB X3D Browser/cobweb/cobweb.js/cobweb/Browser/Core/TextureQuality.js'
In module tree:
    cobweb.js
      cobweb/X3D
        cobweb/Browser/X3DBrowser
          cobweb/Browser/X3DBrowserContext
            cobweb/Browser/Core/X3DCoreContext
              cobweb/Browser/Core/BrowserOptions

    at Error (native)

    at /usr/local/lib/node_modules/jamjs/node_modules/requirejs/bin/r.js:22384:23
    at /usr/local/lib/node_modules/jamjs/node_modules/requirejs/bin/r.js:2683:39
    at /usr/local/lib/node_modules/jamjs/node_modules/requirejs/bin/r.js:2623:25
    at Function.prim.nextTick (/usr/local/lib/node_modules/jamjs/node_modules/requirejs/bin/r.js:22217:9)
    at Object.p.errback (/usr/local/lib/node_modules/jamjs/node_modules/requirejs/bin/r.js:2622:26)
    at Object.p.callback (/usr/local/lib/node_modules/jamjs/node_modules/requirejs/bin/r.js:2608:23)
    at Object.p.promise.then (/usr/local/lib/node_modules/jamjs/node_modules/requirejs/bin/r.js:2662:23)
    at build (/usr/local/lib/node_modules/jamjs/node_modules/requirejs/bin/r.js:22342:12)
    at runBuild (/usr/local/lib/node_modules/jamjs/node_modules/requirejs/bin/r.js:23843:17)
    at Object.context.execCb (/usr/local/lib/node_modules/jamjs/node_modules/requirejs/bin/r.js:1729:33)
Failed
make: *** [all] Error 1
```

**Comment by create3000** [comment]
Cannot reproduce this error.
If you have question to TextureQuality used in Cobeweb feel free to ask this directly!

**Comment by create3000** [comment]
Besides this, the texture quality can be adjusted with scripting: Browser.setBrowserOption("TextureQuatlity", ["Low", "Medium", ""High]) but this is Cobweb specific, and Vincent the texture quality used within Cobweb is well adjusted like other well known Players in the X3D cosmos do it too.

**Comment by vincentmarchetti** [comment]
My problem is that I don't have a source file for TextureQuality.js 
This file is referenced in BrowserOption.js with path
cobweb/Browser/Core/TextureQuality
but I have no such file in my clone  of the master branch, at commit:
commit 6f98602bdcd70661f6c6dc806d3109936ef6f7c7
Author: Holger Seelig <holger.seelig@yahoo.de>
Date:   Sat Dec 12 21:26:27 2015 +0100

I don't see this file in the web browser view at Github, either
Do I need to retrieve this file from another source?

**Comment by vincentmarchetti** [comment]
Hit wrong button, did not intend to close this issue.
Thanks for your help.

**Comment by vincentmarchetti** [comment]
My problem is that I don't have a source file for TextureQuality.js 
This file is referenced in BrowserOption.js with path
cobweb/Browser/Core/TextureQuality
but I have no such file in my clone of the master branch, at commit:
commit 6f98602
Author: Holger Seelig holger.seelig@yahoo.de
Date: Sat Dec 12 21:26:27 2015 +0100

I don't see this file in the web browser view at Github, either
Do I need to retrieve this file from another source?

**Comment by create3000** [comment]
I must really excuse me by yourself, but my version control system from MonoDevelop I used does not always what I want. I have commited the desperately missed file for you now.

**Comment by vincentmarchetti** [comment]
Thank you, that extra file added resolved this problem,

My goal is to contribute toward implementing the nodes in the CADGeometry component, particularly the CADAssembly, CADPart, and CADFace nodes. I am interested in using Cobweb to render the X3D files that are generated by the STEP -> X3D converter at http://spri.kshell.com. An example of an X3D file which uses the CADAssembly/CADPart/CADFace nodes to represent product structure is at http://spri.kshell.com/xt/shape/psghtq40qqvj/shape_2336333638.x3d ; also available as X3DOM page at http://spri.kshell.com/xt/shape/x3dom/psghtq40qqvj/2336333638 . 

Any progress I make I will offer back to the public project.

Vince Marchetti

**Comment by create3000** [comment]
I would very appreciate if you could contribute this to Cobweb. Also a look at https://github.com/create3000/titania/tree/master/libtitania-x3d/Titania/X3D/Components/CADGeometry can give you an advice how to implement this in Cobweb.

**Comment by create3000** [comment]
Fix commited (CADGeometry).

**Comment by vincentmarchetti** [comment]
Your latest commit allows the X3D files that are converted from STEP to be rendered in Cobweb. I will implement this view on the SPRI server (http://spri.kshell.com) and send you links soon.

**Comment by create3000** [comment]
Fix released (Version 1.20).
